packages/salesforcedx-vscode-soql/package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "@salesforce/apex-tmlanguage": "1.6.0",
     "@salesforce/core": "2.28.0",
+    "@types/jsforce": "1.9.40",
```

Comment:
@types/jsforce is already included in this file (line 53).  Let's include it only once.  Also, should this a dependency, or a dev dependdency?

Reply:
good catch.  Yeah it _should_ be a devDep, but @salesforce/core has it as a dependency so in order for us to avoid getting the version they have (which breaks us) we have to have it be a dependency as well.